opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -49,7 +49,7 @@
 
 #include <opm/common/utility/platform_dependent/reenable_warnings.h>
 
-#if HAVE_CUDA
+#if HAVE_CUDA + HAVE_OPENCL
```

Comment:
That's going to be confusing in 6 months' time.  Please either write this as
```C++
#if HAVE_CUDA || HAVE_OPENCL
```
or as
```C++
#if HAVE_CUDA && HAVE_OPENCL
```
depending on what you actually mean.